packages/components/buttons/secondary-button/src/secondary-button.styles.ts

```diff
@@ -124,13 +124,13 @@ const getSizeStyles = (size: TSecondaryButtonProps['size']) => {
     case 'medium':
       return css`
         height: ${designTokens.heightForButtonAsMedium};
-        border-radius: ${designTokens.borderRadiusForButtonAsMedium};
+        border-radius: ${designTokens.borderRadius4};
```

Comment:
I'm wondering if we need this style as both for `medium` and `big` size is the same value and also this is the default value in the `secondary-button.tsx` file below .